great_expectations/rule_based_profiler/domain_builder/categorical_column_domain_builder.py

```diff
@@ -33,6 +37,12 @@ def __init__(
         batch_list: Optional[List[Batch]] = None,
         batch_request: Optional[Union[BatchRequest, RuntimeBatchRequest, dict]] = None,
         data_context: Optional["DataContext"] = None,  # noqa: F821
+        include_semantic_types: Optional[
+            Union[str, SemanticDomainTypes, List[Union[str, SemanticDomainTypes]]]
+        ] = None,
+        exclude_semantic_types: Optional[
+            Union[str, SemanticDomainTypes, List[Union[str, SemanticDomainTypes]]]
+        ] = None,
```

Comment:
We should add some defaults here to exclude columns e.g. unique columns

Reply:
Great idea, @anthonyburdi !!!  Adding (in next commit).